src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -162,50 +164,87 @@ public void receiveEditCards() {
     @Override
     public void receivePostInitialize() {
         declarePacks();
+        BaseMod.logger.info("Full list of packs: " + allPacks.stream().map(pack -> pack.name).collect(Collectors.toList()));
```

Comment:
I added in a few logs like this that were helpful for my debugging. I've left them in for this PR because I think they may be helpful during the development process -- they may help work out any issues that people contributing to this run into, or at least confirm that their pack is being registered. Feel free to remove later if they're no longer helpful.